signac/contrib/hashing.py

```diff
@@ -8,7 +8,18 @@
 
 
 def calc_id(spec):
-    "Calculate and return a hash value for the given spec."
+    """Calculate and return a hash value for the given spec."
+
+    Parameters
+    ----------
+    spec :
+
```

Comment:
Similar to above question about sequences -- this can accept dictionaries, lists, integers, etc. Anything that is JSON-encodable. I don't know what the best recommendation is for a type.
```suggestion
    spec : Collection
        A JSON-encodable collection.
```